packages/website/tsconfig.json

```diff
@@ -31,7 +31,7 @@
     "noFallthroughCasesInSwitch": true,
     "noImplicitOverride": true,
     "noImplicitReturns": true,
-    "noPropertyAccessFromIndexSignature": true,
+    "noPropertyAccessFromIndexSignature": false,
```

Comment:
reset to `true`

Reply:
@fullykubed i think we need to chose one and I've never seen typescript complain about access `process.env` using dot notation. So little lost on why it's doing that. 

<img width="880" alt="image" src="https://github.com/user-attachments/assets/f89feb1c-3e73-40a9-893a-c07cfa7d98c6">

<img width="1618" alt="image" src="https://github.com/user-attachments/assets/c274b37a-f9af-4e6e-b1e9-c8769082fc09">

For now, i've adjusted to include a new `process.env.d.ts` to define this.